python/shotgun_desktop/upgrade_startup.py

```diff
@@ -156,10 +156,10 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
         # know something wrong is going on.
         logger.exception("Unexpected error when updating startup code.")
         DesktopMessageBox.critical(
-            "ShotGrid Desktop update failed",
-            "There is a new update of the ShotGrid Desktop, but it couldn't be installed. ShotGrid "
+            "Flow Production Tracking update failed",
+            "There is a new update of the Flow Production Tracking, but it couldn't be installed. Flow Production Tracking "
             "Desktop will be launched with the currently installed version of the code.\n"
-            "If this problem persists, please <a href='%s'>contact</a> ShotGrid support.\n"
+            "If this problem persists, please <a href='%s'>contact</a> Flow Production Tracking support.\n"
```

Comment:
There is a new update of the PTR desktop component, but it couldn't be installed.
The PTR desktop app will be launched with the currently installed version of the code.